ui/admin-portal/src/app/components/candidates/view/special-links/view-candidate-special-links.component.ts

```diff
@@ -40,21 +40,6 @@ export class ViewCandidateSpecialLinksComponent implements OnInit, OnChanges {
   ngOnInit() {
   }
 
-  ngOnChanges(changes: SimpleChanges): void {
```

Comment:
Remove this code, not longer necessary api call. Fetch the candidate once in parent tab component